libraries/AP_RAMTRON/AP_RAMTRON.cpp

```diff
@@ -87,6 +95,13 @@ bool AP_RAMTRON::init(void)
                 id = i;
                 break;
             }
+        } else if (ramtron_ids[i].rdid_type == RDID_type::Petabytes) {
+            const petabytes_rdid *petabytes = (const petabytes_rdid *)rdid;
```

Comment:
can you also check the manufacturer[] bytes?

Reply:
> can you also check the manufacturer[] bytes?

the manufacturer[] bytes are two bytes. Petabytes ID is 0A7F different form Fujitsu's 047F.